index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>BestShop</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&family=Raleway:wght@300;400;700;800&display=swap" rel="stylesheet">
+    <link href="css/main.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="container">
+            <span class="logo_text">BestShop</span>
+            <input type="checkbox" class="menu__btn" id="menu__btn">
+            <label for="menu__btn" class="menu__toggle">
+                <span></span>
+                <span></span>
+                <span></span>
+            </label>
+            <nav class="nav_list">
+                <ul class="nav_list">
+                    <li><a class="nav_link" href="">WHY US</a></li>
+                    <li><a class="nav_link" href="">WBENEFITS</a></li>
+                    <li><a class="nav_link" href="">PRICES</a></li>
+                    <li><a class="nav_link" href="">CONTACT</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <section class="section_wellcome">
+        <img class="background_wellcome" src="../assets/Background.png" alt="Biurko na którym jest laptop, kawa, telefon, notatnik i karty do gry">
+        <h1 class="headline_wellcome">Sell More!</h1>
```

Comment:
nie wydaje mi się że to nnajważninejszy tekst na stronie. użyłaś h1, która wskazuje na najważniejszy tekst na stronie